docs/recipes/install/common/openchami_host_configure.tex

```diff
@@ -0,0 +1,23 @@
+% -*- mode: latex; fill-column: 120; -*-
+To begin, enable use of the OpenHPC repository by adding it to the local list of available package repositories.
```

Comment:
This file seems to be unused and I am also confused why it exist. This all seems to be information from somewhere else.